OneSignal.Android.Binding/OneSignal.Android.Binding.csproj

```diff
@@ -10,7 +10,7 @@
     <MonoAndroidResourcePrefix>Resources</MonoAndroidResourcePrefix>
     <MonoAndroidAssetsPrefix>Assets</MonoAndroidAssetsPrefix>
     <AssemblyName>OneSignal.Android.Binding</AssemblyName>
-    <TargetFrameworkVersion>v11.0</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v12.0</TargetFrameworkVersion>
```

Comment:
upgrading to v12 was required because i referenced the nuget Xamarin.AndroidX.Core which was only supporting v12. (and followings maybe?)

Reply:
When updating the TargetFrameworkVersion here I think we'll also need to update it in [OneSignal.Xamarin.Android](https://github.com/OneSignal/OneSignal-Xamarin-SDK/blob/main/OneSignalSDK.Xamarin.Android/OneSignalSDK.Xamarin.Android.csproj) and [OneSignalApp.Sample.Droid](https://github.com/OneSignal/OneSignal-Xamarin-SDK/blob/main/Samples/OneSignalApp.Sample.Droid/OneSignalApp.Sample.Droid.csproj).